index.js

```diff
@@ -5,7 +5,8 @@ import {fileURLToPath} from 'node:url';
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-const exec = (command, arguments_, shell) => execFileSync(command, arguments_, {encoding: 'utf8', shell}).trim();
+const exec = (command, arguments_, shell) =>
+	execFileSync(command, arguments_, {encoding: 'utf8', shell, stdio: ['pipe', 'pipe', 'ignore']}).trim();
```

Comment:
```suggestion
	execFileSync(command, arguments_, {encoding: 'utf8', shell, stdio: ['ignore', 'pipe', 'ignore']}).trim();
```

For readability? Since we don't use stdin either.